data/mods/ssb/moves.js

```diff
@@ -809,6 +809,49 @@ let BattleMovedex = {
 		target: "normal",
 		type: "Normal",
 	},
+	// Birdy~!
+	justdance: {
+		accuracy: true,
+		basePower: 0,
+		category: "Status",
+		desc: "Uses a random dance move twice in a row.",
+		shortDesc: "Uses a random dance move twice in a row.",
+		id: 'justdance',
+		name: "Just Dance",
+		isNonstandard: "Custom",
+		pp: 5,
+		priority: 0,
+		flags: {dance: 1},
+		onTryMove(pokemon) {
+			this.attrLastMove('[still]');
+		},
+		onHit(target, source, effect) {
+			const moves = [];
+			for (const i in exports.BattleMovedex) {
```

Comment:
You can't loop through this here, since `exports.BattleMovedex` in this context will only be the SSB custom moves. Instead loop through `this.data.Movedex` (see glitchout for more info).